recipe/meta.yaml

```
@@ -11,7 +11,7 @@ source:
   sha256: 7cbcd1400c18b908aea8be0cb9fcb6305500ff29737f67db456ff607308af8e4
 
 build:
-  number: 2
+  number: 3
   skip: True  # [cuda_compiler_version == "None" or cuda_compiler_version == "10.2"]
```

As conda-forge is now on CUDA 11.2+ ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1708 ), the skip for CUDA 10.2 can be dropped

```suggestion
  skip: True  # [cuda_compiler_version == "None"]
```